test/command/suite/sharding/logical_range_filter/columns/stage/filtered/window_function/day/max.expected

```diff
@@ -86,5 +86,754 @@ logical_range_filter Logs   --max '2017/03/16 08:22:00'   --max_border include
+#:000000000000000 filter(0): between(#<accessor _key(anonymous).timestamp(Logs_20170316)>, "2017/03/01 13:00:00", "include", "2017/03/01 14:00:00", "exclude")
+#:000000000000000 filter(0): between(#<accessor _key(anonymous).timestamp(Logs_20170316)>, "2017/03/01 14:00:00", "include", "2017/03/01 15:00:00", "exclude")
+#:000000000000000 filter(0): between(#<accessor _key(anonymous).timestamp(Logs_20170316)>, "2017/03/01 15:00:00", "include", "2017/03/01 16:00:00", "exclude")
+#:000000000000000 filter(0): between(#<accessor _key(anonymous).timestamp(Logs_20170316)>, "2017/03/01 16:00:00", "include", "2017/03/01 17:00:00", "exclude")
+#:000000000000000 filter(0): between(#<accessor _key(anonymous).timestamp(Logs_20170316)>, "2017/03/01 17:00:00", "include", "2017/03/01 18:00:00", "exclude")
+#:000000000000000 filter(0): between(#<accessor _key(anonymous).timestamp(Logs_20170316)>, "2017/03/01 18:00:00", "include", "2017/03/01 19:00:00", "exclude")
+#:000000000000000 filter(0): between(#<accessor _key(anonymous).timestamp(Logs_20170316)>, "2017/03/01 19:00:00", "include", "2017/03/01 20:00:00", "exclude")
+#:000000000000000 filter(0): between(#<accessor _key(anonymous).timestamp(Logs_20170316)>, "2017/03/01 20:00:00", "include", "2017/03/01 21:00:00", "exclude")
+#:000000000000000 filter(0): between(#<accessor _key(anonymous).timestamp(Logs_20170316)>, "2017/03/01 21:00:00", "include", "2017/03/01 22:00:00", "exclude")
+#:000000000000000 filter(0): between(#<accessor _key(anonymous).timestamp(Logs_20170316)>, "2017/03/01 22:00:00", "include", "2017/03/01 23:00:00", "exclude")
+#:000000000000000 filter(0): between(#<accessor _key(anonymous).timestamp(Logs_20170316)>, "2017/03/01 23:00:00", "include", "2017/03/02 00:00:00", "exclude")
+#:000000000000000 filter(0): between(#<accessor _key(anonymous).timestamp(Logs_20170316)>, "2017/03/02 00:00:00", "include", "2017/03/02 01:00:00", "exclude")
+#:000000000000000 filter(0): between(#<accessor _key(anonymous).timestamp(Logs_20170316)>, "2017/03/02 01:00:00", "include", "2017/03/02 02:00:00", "exclude")
+#:000000000000000 filter(0): between(#<accessor _key(anonymous).timestamp(Logs_20170316)>, "2017/03/02 02:00:00", "include", "2017/03/02 03:00:00", "exclude")
+#:000000000000000 filter(0): between(#<accessor _key(anonymous).timestamp(Logs_20170316)>, "2017/03/02 03:00:00", "include", "2017/03/02 04:00:00", "exclude")
+#:000000000000000 filter(0): between(#<accessor _key(anonymous).timestamp(Logs_20170316)>, "2017/03/02 04:00:00", "include", "2017/03/02 05:00:00", "exclude")
+#:000000000000000 filter(0): between(#<accessor _key(anonymous).timestamp(Logs_20170316)>, "2017/03/02 05:00:00", "include", "2017/03/02 06:00:00", "exclude")
+#:000000000000000 filter(0): between(#<accessor _key(anonymous).timestamp(Logs_20170316)>, "2017/03/02 06:00:00", "include", "2017/03/02 07:00:00", "exclude")
+#:0000000000000
```

Comment:
In my opinion, too many output logs.
Therefore, we are afraid that this output influence performance.

Is not there an influence in performance?

Reply:
Oh, It may affects.

Reply:
It is generated by table.select(expression) in filter_by_range.